tools/docker-build/README.md

```diff
@@ -0,0 +1,22 @@
+# Building CircuitPython with Docker
+
+On a Linux machine with `docker` installed, an executable for any supported CircuitPython board can be built without setup by invoking the command `sudo tools/docker-build/build <port> <board-name> <option>` from the CircuitPython root directory.  For example, if the CircuitPython repo is located at `~/circuitpython` the commands
+
+```bash
+    cd ~/circuitpython
+    sudo tools/docker-build/build espressif adafruit_feather_esp32s2_reverse_tft
+    sudo tools/docker-build/build raspberrypi waveshare_rp2040_zero
+    sudo tools/docker-build/build atmel-samd grandcentral_m4_express DEBUG=1
```

Comment:
Is it typical that docker usage runs things as root? It seems like the whole build will be run as root in the docker container.

Reply:
Yeah this is one of the things that could probably be improved.  The container doesn't have to be run with `sudo` but the processes inside the container run as `root` so the owner of anything created by the container ends up as `root` which makes `git` unhappy.  The reason I'm having it run as `sudo` is so that I can patch this up after the fact and change the owner back to who it should be.